### ISSUE TITLE
Add a blacklist to the config

### ISSUE THREAD

**Issue by BigPhobos** [AlexModGuy/RetroDamageIndicators#7]
Currently there is no configurable mob blacklist. This feature would be useful for modpack development.

**Comment by polykatt** [comment]
I agree this would be great to have. Also an option to disable displaying bosses hud by default

**Comment by barbod5479** [comment]
I agree with this because I'm making a medieval modpack and I need this feature for my modpack
PLEASE ADD THIS!
